DuckDuckGo/TabViewControllerLongPressMenuExtension.swift

```diff
@@ -66,7 +66,7 @@ extension TabViewController {
     
     private func onReadingAction(forUrl url: URL) {
         Pixel.fire(pixel: .longPressMenuReadingListItem)
-        try? SSReadingList.default()?.addItem(with: url, title: nil, previewText: nil)
+        ((try? SSReadingList.default()?.addItem(with: url, title: nil, previewText: nil)) as ()??)
```

Comment:
Optional: Is there a way to clean the syntax up here? It's not very readable. Not a blocker, if it doesn't work out I'll give it a try when I upgrade XCode too.

Reply:
Xcode made this change, but let me have a look.

Reply:
I reverted it - doesn't see to be a problem.